ProjectK.Business/ExperiencePoints/IExperienceRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    public interface IExperienceRepository
+    {
+        Task<Experience> GetCharacterXpToLevel(int charId);
```

Comment:
Method names for methods that return Tasks should always end with `Async` suffix !!!

Fix this everywhere!!!

Reply:
Ok. Added it to every one